docs/release-notes/index.md

```diff
@@ -18,7 +18,7 @@ For product updates that are in progress or planned [visit the Codacy public roa
 
 2023
 
--   [New Coverage engine status checks November 23, 2023](cloud/cloud-2023-11-23-new-coverage-engine-status-checks.md)
+-   [New Coverage engine November 23, 2023](cloud/cloud-2023-11-23-new-coverage-engine-status-checks.md)
```

Comment:
Note: It seems that Vale is not very fond of "rollout", though it's a [valid word](https://www.merriam-webster.com/dictionary/rollout)... 🤔 

```suggestion
-   [Rollout of new Coverage engine November 23, 2023](cloud/cloud-2023-11-23-new-coverage-engine-status-checks.md)
```